tmk_core/chibios.mk

```
@@ -235,28 +235,18 @@ qmk: $(BUILD_DIR)/$(TARGET).bin
 	printf "@ $(TARGET).json\n@=info.json\n" | zipnote -w $(TARGET).qmk
 
 define EXEC_DFU_UTIL
+	until $(DFU_UTIL) -l | grep -q "Found DFU"; do\
+		echo "Trying again in 5s." ;\
```

For sanity's sake, could we make this consistent with `:dfu`? 

In fact, we should probably move this to messages.mk...  (but maybe later).
```suggestion
		echo "Error: Bootloader not found. Trying again in 5s." ;\
```